docs/changelog.md

```diff
@@ -42,6 +42,11 @@ permalink: /changelog/
 * **support-ktx**
   * Add `Activity.reportFullyDrawnSafe`, a function to call `Activity.reportFullyDrawn` while catching crashes under some circumstances.
 
+* **browser-tabstray**
```

Comment:
I need to update the changelog to put this into the next release before landing.